README.md

```diff
@@ -47,6 +53,15 @@ docker run \
 security-sensitive API, so only expose it beyond 127.0.0.1 if you know what 
 you're doing.
 
+By default the container uses the following defaults:  
+- `SIA_DIR` is `/sia`
```

Comment:
I think container users cannot see this value unless they build an image by themselves because `SIA_DIR` is a build argument. It, hence, looks a bit confusing for me (at least `-e` option won't work for this variable).

Reply:
Yes, you cannot override this but you can mount a volume onto it. So container users can run the container with `--volume /users/myuser/mysiadir:/sia` and that will allow them to attach their own external directory on the container's `/sia` path. This comment is meant to tell people where to attach their volume, should they want to do that.

The idea is that you don't really need to modify this path because you can modify the other two, so you can arrange them in any way you want relative to this one.

Reply:
I meant that these sentences suggest you can modify `SIA_DIR` with `-e` option and it's confusing:

```
By default the container uses the following defaults:  
- `SIA_DIR` is `/sia`
- `SIA_DATA_DIR` is `/sia-data`
- `SIAD_DATA_DIR` is `/sia-data`

You can change those defaults by supplying new environment variables for the 
container using the `-e` option:  
`-e SIAD_DATA_DIR=/new-siad-data-dir`
```

I also think that mounting a local directory onto `/sia` wouldn't be practical because `siad` and other scripts in that directory will be overwritten.

Reply:
I see. The documentation can be more explicit. I made some changes.

Reply:
Looks nice. Thanks!